State Validator fails with "No config data record available of type 'class com.hedera.node.config.data.BlockRecordStreamConfig'"

Validator of recent main (https://github.com/hiero-ledger/hiero-consensus-node/commit/8049a3f42a9960060f73ee1b2d2aed910da7ee8f) failed:


```
java.lang.RuntimeException: java.lang.IllegalArgumentException: No config data record available of type 'class com.hedera.node.config.data.BlockRecordStreamConfig'
	at com.hedera.statevalidation.util.StateUtils.initState(StateUtils.java:204)
	at com.hedera.statevalidation.util.StateUtils.getState(StateUtils.java:132)
	at com.hedera.statevalidation.util.StateUtils.getDefaultState(StateUtils.java:122)
	at com.hedera.statevalidation.ValidateCommand.call(ValidateCommand.java:119)
	at com.hedera.statevalidation.ValidateCommand.call(ValidateCommand.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at com.hedera.statevalidation.StateOperatorCommand.main(StateOperatorCommand.java:48)
Caused by: java.lang.IllegalArgumentException: No config data record available of type 'class com.hedera.node.config.data.BlockRecordStreamConfig'
	at com.swirlds.config.impl.internal.ConfigDataService.getConfigData(ConfigDataService.java:96)
	at com.swirlds.config.impl.internal.ConfigurationImpl.getConfigData(ConfigurationImpl.java:233)
	at com.hedera.node.app.records.schemas.V0740BlockRecordSchema.restart(V0740BlockRecordSchema.java:34)
	at com.hedera.node.app.state.merkle.MerkleSchemaRegistry.migrate(MerkleSchemaRegistry.java:214)
	at com.hedera.node.app.services.OrderedServiceMigrator.lambda$doMigrations$0(OrderedServiceMigrator.java:85)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at java.base/java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1118)
	at com.hedera.node.app.services.OrderedServiceMigrator.doMigrations(OrderedServiceMigrator.java:78)
	at com.hedera.statevalidation.util.StateUtils.initServiceMigrator(StateUtils.java:300)
	at com.hedera.statevalidation.util.StateUtils.initState(StateUtils.java:201)
	... 13 more
```


Logs:
https://perf.analytics.eng.hashgraph.io/ephemeral/test_runs/24193-adjust-performance-baseline-in-MQPT-SDPT_AdHoc7_728/report/validator/

--- a/hedera-state-validator/src/main/java/com/hedera/statevalidation/util/ConfigUtils.java
+++ b/hedera-state-validator/src/main/java/com/hedera/statevalidation/util/ConfigUtils.java
@@ -22,7 +22,9 @@ import com.hedera.node.config.converter.ScaleFactorConverter;
 import com.hedera.node.config.converter.SemanticVersionConverter;
 import com.hedera.node.config.data.AccountsConfig;
 import com.hedera.node.config.data.ApiPermissionConfig;
+import com.hedera.node.config.data.BlockRecordStreamConfig;
 import com.hedera.node.config.data.BlockStreamConfig;
+import com.hedera.node.config.data.BlockStreamJumpstartConfig;
 import com.hedera.node.config.data.BootstrapConfig;
 import com.hedera.node.config.data.FilesConfig;
 import com.hedera.node.config.data.HederaConfig;
@@ -115,6 +117,8 @@ public final class ConfigUtils {
                 .withConfigDataType(PcesConfig.class)
                 .withConfigDataType(BasicConfig.class)
                 .withConfigDataType(MetricsConfig.class)
+                .withConfigDataType(BlockRecordStreamConfig.class)
+                .withConfigDataType(BlockStreamJumpstartConfig.class)
                 .withSource(new SimpleConfigSource().withValue("merkleDb.usePbj", false))
                 .withSource(new SimpleConfigSource().withValue("merkleDb.minNumberOfFilesInCompaction", 2))
                 .withSource(new SimpleConfigSource().withValue("merkleDb.maxFileChannelsPerFileReader", FILE_CHANNELS))

